### PR TITLE
add new tables and columns to the database

### PR DESCRIPTION
**Pull request by azum4roll**
Build check done
Start game check done
Now fixing inevitable Lua errors with all those BOOLEAN columns changing to boolean.

### CODE REVIEW

**Comment by azum4roll** [comment]
Test run as observer done, should be safe to merge.

File: CvGameCoreDLL_Expansion2/Lua/CvLuaUnit.cpp [hunk 1]
```diff
@@ -1811,23 +1811,6 @@ int CvLuaUnit::lIsRangeAttackOnlyInDomain(lua_State* L)
 	return 1;
 }
 
-//------------------------------------------------------------------------------
-int CvLuaUnit::lIsCoastalAttackOnly(lua_State* L)
```
Comment (KungCheops) [inline CvGameCoreDLL_Expansion2/Lua/CvLuaUnit.cpp:1815]: It's not possible to build the dll after you removed this (forgot to update line 338). Looks like there's a reference to this in Bombardment.lua as well.

Comment (azum4roll) [reply CvGameCoreDLL_Expansion2/Lua/CvLuaUnit.cpp:1815]: It shouldn't affect build, only having error when Bombardment.lua is run (when a human selects a city to attack, which I didn't test).

Comment (azum4roll) [reply CvGameCoreDLL_Expansion2/Lua/CvLuaUnit.cpp:1815]: fixed in latest commit

**Pull request merged**